src/Microsoft.ML.Data/Scorers/PredictionTransformer.cs

```diff
@@ -74,16 +69,72 @@ internal PredictionTransformerBase(IHost host, ModelLoadContext ctx)
             ms.Position = 0;
             var loader = new BinaryLoader(host, new BinaryLoader.Arguments(), ms);
             TrainSchema = loader.Schema;
+        }
+
+        public abstract ISchema GetOutputSchema(ISchema inputSchema);
+
+        /// <summary>
+        /// Transforms the input data.
+        /// </summary>
+        /// <param name="input"></param>
+        /// <returns></returns>
+        public abstract IDataView Transform(IDataView input);
 
+        protected void SaveModel(ModelSaveContext ctx)
+        {
+            // *** Binary format ***
```

Comment:
>*** Binary format *** [](start = 15, length = 21)

whenever you save or load, need `*** Binary format ***`

Reply:
?

---
In reply to: [218633828](https://github.com/dotnet/machinelearning/pull/912#discussion_r218633828) [](ancestors = 218633828)